api/README.md

```diff
@@ -147,7 +147,7 @@ Specifically, update `info@ovio.org` with a domain admin email and `prism-api.ov
 To deploy, run:
 
 ```
-make deploy
+source set_envs.sh && make deploy
```

Comment:
Could the `source...` part be moved to the `Makefile` instead? I find it nice when all the useful commands are available in there with tab completion. And it's slightly less error-prone.

Reply:
Also noting that the text above (line 145) is outdated as the variable definition moved elsewhere.

Reply:
Fixed ✅